dChatServer/PlayerContainer.h

```diff
@@ -7,6 +7,9 @@
 #include "dServer.h"
 #include <unordered_map>
 
+#include <nlohmann/json.hpp>
+using json = nlohmann::json;
```

Comment:
This should be moved to cpp files.  Using Using in a header is bad practice and pollutes the namespace

Reply:
I have an idea about something to help with this an only need to do this once

Reply:
lmk if this solution I have pushed is better

Reply:
this is doing the same thing just moved to a second file

each file that uses json should include the header manually and the cpp file can do `using json = nlohmann::json;`.

Reply:
a #include just takes the contents of the included file and pastes it right there, so moving it to another header just moves the work

Reply:
the purpose of unnamed namespaces is to make it to the contents within are guaranteed to be the ones in this file.  placing an unnamed namespace in a header just creates a new version of its contents in each file that includes it, likely not what was intended

Reply:
Yeah, that makes sense, honestly don't even really need the using. So I may just not do that at all

Reply:
its fine to use it but only in a cpp file

Reply:
Would it not be inconsistent between the h and cpp then? I'd rather it be consistent

Reply:
its not inconsistent though, in one you use the fully qualified name to prevent namespace pollution, and in the implementation you can use a shorthand.  for example, one could do
```cpp
namespace f {
    void foo();
}

void f::foo() {};
// or
namespace f {
    void foo() {}
};
```
i see it as perfectly fine and reasonable to use fully qualified name for the return in the header and the shorthand for the return and usage in the implementation.

Reply:
hmmmm okay